### ISSUE TITLE
Make a demo

### ISSUE THREAD

**Issue by liurui16** [yysijie/st-gcn#36]
Thanks for your great work!Recently,I'm trying to make a demo of your work.
How can I input one .skeleton file from **NTU RGB+D** dataset and get its predicted label?
Do you have any these kind of script or just inform me some tips?
Thank you very much!

**Comment by LinCh83** [comment]
Hi
I want to do the same thing like you.
But I didn't find it
Can you tell me where you found it?
Thank you very much!

**Comment by Sindreedelange** [comment]
@liurui16 - I am pursuing something similar to you, and I am not able to find the Softmax that they are referring to in the paper. 

From what I can tell, the last layer is a Conv2d layer, with a 7 value output, which is fed to the CrossEntropy Loss function.
![image](https://user-images.githubusercontent.com/9415625/52843933-be5dec00-3103-11e9-892d-2407e19c0d37.png)

**Comment by JasOlean** [comment]
> I found it.
> I really learnt a lot from your work!

Where did you find Softmax in code?

**Comment by ouss3ma** [comment]
Does any one find the  Softmax in code?

**Comment by jbanaszkiewicz** [comment]
I also can't find softmax anywhere in the code. Does someone know where is it?

**Comment by Infin1te2021** [comment]
It doesn't matter. Although the last layer is a Fully Connected layer, we use the [Cross Entropy Loss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html), which can be taken as a [LogSoftmax](https://pytorch.org/docs/stable/generated/torch.nn.LogSoftmax.html#torch.nn.LogSoftmax) followed by [NLLLoss](https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss).
